src/System.Buffers/ref/System.Buffers.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Buffers
+{
+    public abstract class ArrayPool<T>
+    {
+        public const int DefaultNumberOfArraysPerBucket = 50;
+        public const int DefaultArrayLength = 1024000;
```

Comment:
The value was just a debugging approximation that I used...changed to be the actual value of 1MB :). I also made the fixes from your comment to make a parameterless Create call and one without default parameters, allowing the constants to be internal